mentat/code_file_manager.py

```diff
@@ -86,7 +95,7 @@ def write_changes_to_files(
 
             if not file_edit.is_creation:
                 stored_lines = self.file_lines[rel_path]
-                if stored_lines != self.read_file(rel_path):
+                if stored_lines != _read_file(self.config, rel_path):
```

Comment:
This is where we actually check for those changes, but now that the get method for file_lines just calls _read_file anyways, this will never detect any changes